views/engineering_garage/edit_teammates.erb

```diff
@@ -0,0 +1,67 @@
+<h1 class="dcf-txt-h3" id="Edit_Teammates">Edit Teammates</h1>
+<div class="dcf-grid-thirds dcf-col-gap-vw">
+<div>
+    <a href="/checkout/project/<%= project_id %>/edit" class="dcf-btn dcf-btn-secondary dcf-25% dcf-w-max-2xs" type="button">Return to Edit Project</a>
+</div>
+<div></div>
+    <form class="dcf-form dcf-w-100%" id="find-nuid" action="/checkout/project/<%= project_id %>/edit/teammates/?" method="POST" enctype="application/x-www-form-urlencoded">
+        <label for="nuid-input">Add a Teammate by NUID</label>
+        <div class="dcf-input-group">
+            <input id="nuid-input" name="nuid">
+            <button class="dcf-btn dcf-btn-primary" type="submit">Add Teammate</button>
+        </div>
+    </form>
+</div>
+<fieldset class="dcf-mt-4">
+    <legend>Project Teammates</legend>
+    <div class="dcf-overflow-x-auto" tabindex="0">
+        <% if teammates.count > 0 %>
+        <table class="dcf-table dcf-table-striped dcf-w-100% dcf-table-responsive dcf-mb-6" aria-describedby="teammate-list-table">
+            <thead>
+                <tr>
+                    <th scope="col">Name</th>
+                    <th scope="col">Email</th>
+                    <th scope="col">Date Added</th>
+                    <th scope="col">Teammate Actions</th>
+                </tr>
+            </thead>
+            <tbody>
+            <% teammates.each do | teammate | %>
+                <% user = User.find_by(id: teammate.teammate_id) %>
+                <tr>
+                    <td><%= user.full_name %></td>
+                    <td><%= user.email %></td>
+                    <td><%= teammate.created_on.strftime('%m/%d/%Y') %></td>
+                    <td class="dcf-d-flex dcf-flex-row-rev dcf-flex-shrink-1 dcf-gap-2">
+                        <button class="dcf-btn-toggle-modal dcf-btn dcf-btn-primary" data-toggles-modal="remove-<%= teammate[:id] %>" type="button">Remove</button>
```

Comment:
I wouldn't have this open a modal. I would replace the button/modal with just a normal form with the button being the submit button to the form. Then I would add a `onsubmit` for the "Are you sure?" thing. That is what we do for a lot of our apps and its better UX if things that look the same act the same(I think their is a law of design about that or something).

```html
<form onsubmit="return confirm('Do you really want to submit the form?');">
```